### ISSUE TITLE
Lost data when rebuilding index

### ISSUE THREAD

**Issue by Amber1990Zhang** [vesoft-inc/nebula#2206]
I have 19 records.
```
(user@nebula) [NBA]> GO FROM 100,101,102,103,104,105,106,107,108,109,110 OVER follow YIELD follow._src, follow._dst,follow.degree
=============================================
| follow._src | follow._dst | follow.degree |
=============================================
| 100         | 101         | 95            |
---------------------------------------------
| 100         | 102         | 91            |
---------------------------------------------
| 100         | 106         | 90            |
---------------------------------------------
| 101         | 100         | 95            |
---------------------------------------------
| 101         | 102         | 91            |
---------------------------------------------
| 102         | 101         | 75            |
---------------------------------------------
| 103         | 102         | 70            |
---------------------------------------------
| 104         | 103         | 50            |
---------------------------------------------
| 104         | 105         | 60            |
---------------------------------------------
| 105         | 104         | 83            |
---------------------------------------------
| 105         | 110         | 87            |
---------------------------------------------
| 106         | 100         | 88            |
---------------------------------------------
| 106         | 107         | 81            |
---------------------------------------------
| 107         | 106         | 92            |
---------------------------------------------
| 107         | 108         | 97            |
---------------------------------------------
| 108         | 109         | 95            |
---------------------------------------------
| 109         | 110         | 78            |
---------------------------------------------
| 110         | 105         | 85            |
---------------------------------------------
| 110         | 109         | 72            |
---------------------------------------------
Got 19 rows (Time spent: 2.463/4.873 ms)
```
![image](https://user-images.githubusercontent.com/42762957/86435662-0bf7ea80-bd33-11ea-944a-2af9ca63c287.png)
![image](https://user-images.githubusercontent.com/42762957/86435672-131ef880-bd33-11ea-9acb-0f076c357db6.png)

But when I query with `LOOKUP`, only 11 are returned. 8 lost.
![image](https://user-images.githubusercontent.com/42762957/86435624-fd113800-bd32-11ea-97dd-e22e8622ea9d.png)

Dataset link: https://oss-cdn.nebula-graph.io/doc/example_data.zip

**Comment by whitewum** [comment]
is this data loss bug fixed in 1.0.1?

**Comment by CPWstatic** [comment]
As far as i know, this issue is not reproduced in latest version of nebula1.0 and nebula2.0. I'll close it firstly.
